cqlsh_tests/test_cqlsh.py

```diff
@@ -599,7 +599,7 @@ def test_unicode_invalid_request_error(self):
             err = node1.run_cqlsh(cmds=cmd, cqlsh_options=options).stderr
 
         if self.cluster.version() >= LooseVersion('4.0'):
-            assert "Keyspace name must not be empty, more than 48 characters long, or contain non-alphanumeric-underscore characters (got 'ä')" in err
+            assert 'Table name must not be empty or not contain non-alphanumeric-underscore characters (got "ä")' in err
```

Comment:
Should we have a keyspace name here?
```suggestion
            assert 'Keyspace name must not be empty or not contain non-alphanumeric-underscore characters (got "ä")' in err
```

Reply:
Good catch! Sorry that I forgot to check this PR.